root/release_group/ReleaseGroupIndex.js

```diff
@@ -0,0 +1,213 @@
+  from '../static/scripts/common/components/WikipediaExtract';
+import CritiqueBrainzLinks from '../components/CritiqueBrainzLinks';
+import CritiqueBrainzReview
+  from '../static/scripts/common/components/CritiqueBrainzReview';
+import PaginatedResults from '../components/PaginatedResults';
+import TaggerIcon from '../static/scripts/common/components/TaggerIcon';
+import loopParity from '../utility/loopParity';
+import EntityLink from '../static/scripts/common/components/EntityLink';
+import FormRow from '../components/FormRow';
+import FormSubmit from '../components/FormSubmit';
+import Relationships from '../components/Relationships';
+import ReleaseDates from '../components/ReleaseDates';
+import ReleaseCountries from '../components/ReleaseCountries';
+import ReleaseLabelList from '../components/ReleaseLabelList';
+import ReleaseCatnoList from '../components/ReleaseCatnoList';
+import formatBarcode from '../static/scripts/common/utility/formatBarcode';
+import * as manifest from '../static/manifest';
+
+import ReleaseGroupLayout from './ReleaseGroupLayout';
+
+type Props = {|
+  +$c: CatalystContextT,
+  +eligibleForCleanup: boolean,
+  +mostPopularReview: CritiqueBrainzReviewT,
+  +mostRecentReview: CritiqueBrainzReviewT,
+  +numberOfRevisions: number,
+  +pager: PagerT,
+  +releaseGroup: ReleaseGroupT,
+  +releases: $ReadOnlyArray<$ReadOnlyArray<ReleaseT>>,
+  +wikipediaExtract: WikipediaExtractT,
+|};
+
+function buildReleaseStatusTable(releaseStatusGroup) {
+  const status = releaseStatusGroup[0].status;
+  return (
+    <>
+      <tr className="subh">
+        <CatalystContext.Consumer>
+          {($c: CatalystContextT) => (
+            $c.user_exists
+              ? <th />
+              : null
+          )}
+        </CatalystContext.Consumer>
+        <CatalystContext.Consumer>
+          {($c: CatalystContextT) => (
+            <th colSpan={$c.session && $c.session.tport ? 9 : 8}>
+              {status && status.name
+                ? lp_attributes(status.name, 'release_status')
+                : l('(unknown)')}
+            </th>
+          )}
+        </CatalystContext.Consumer>
+      </tr>
+      {releaseStatusGroup.map((release, index) => (
+        <tr className={loopParity(index)} key={release.id}>
+          <CatalystContext.Consumer>
+            {($c: CatalystContextT) => (
+              $c.user_exists
+                ? (
+                  <td>
+                    <input
+                      name="add-to-merge"
+                      type="checkbox"
+                      value={release.id}
+                    />
+                  </td>
+                ) : null
+            )}
+          </CatalystContext.Consumer>
+          <td>
+            <EntityLink entity={release} />
+          </td>
+          <td>{release.combined_format_name || l('[missing media]')}</td>
+          <td>{release.combined_track_count || '-'}</td>
+          <td>
+            <ReleaseDates events={release.events} />
```

Comment:
Not sure why, but the ul inside the `ReleaseDates` component is missing the "nowrap" class (which the TT macro had), so the dates get squished in the table.

Reply:
Added the class